gtf/annotation.go

```diff
@@ -39,7 +40,7 @@ func cDnaToString(v *vcfEffectPrediction, seq map[string][]dna.Base) string {
 func nonCodingToString(v *vcfEffectPrediction, seq map[string][]dna.Base) string {
 	var answer string = v.RefId + ":c."
 	ref := dna.StringToBases(v.Ref)
-	alt := dna.StringToBases(v.Alt)
+	alt := dna.StringToBases(v.Alt[0])//TODO: (riley) This looks like it was written for biallelic positions originally, I have doubled down on this for now but polyallelic bases should be handled.
```

Comment:
Should we also add this expectation (biallelic only) to the main function comment?